torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -740,9 +749,19 @@ def fw_fn(*primals):
     def metadata_fn(*primals):
         return inner_fn(fw_only, primals, use_trace_joint=False)
 
-    args_unwrapped = unwrap_tensor_subclasses(
-        args, is_joint_structure=is_joint_structure
-    )
+    if is_joint_structure:
```

Comment:
manually handling the `is_joint_structure` here sounds good to me, let's just kill the helper and do it everywhere